### PR TITLE
MAGE-42_split_export

### PR DESCRIPTION
**Pull request by TheKeymaster**

### CODE REVIEW

File: Findologic/Search/Controller/Export/ExportController.php [hunk 1]
```diff
@@ -104,8 +104,16 @@ public function execute()
                 ->setContents('Run "composer require findologic/libflexport" in your project directory');
         }
 
-        $xmlExample = new XmlExample();
-        $xml = $xmlExample->createExport();
+        if (!file_exists(__DIR__ . '/../../../Export/Export.php')) {
+            // TODO: Add the link to the export plugin.
```
Comment (dblaichinger) [inline Findologic/Search/Controller/Export/ExportController.php:108]: Add link when export plugin has been uploaded

File: Findologic/Search/Controller/Export/ExportController.php [hunk 2]
```diff
@@ -104,8 +104,16 @@ public function execute()
...
+            return $this->rawResponse
+                ->setHeader('Content-type', 'text/plain')
+                ->setContents('Please install the export plugin. You can find it here http://something');
+        }
+
+        require_once __DIR__ . '/../../../Export/Export.php';
```
Comment (dblaichinger) [inline Findologic/Search/Controller/Export/ExportController.php:114]: Is the base plugin and the export plugin in one Findologic folder?

File: Findologic/Search/Controller/Export/ExportController.php [hunk 3]
```diff
@@ -3,7 +3,7 @@
 namespace Findologic\Search\Controller\Export;
 
 use FINDOLOGIC\Export\Exporter;
-use Findologic\Search\Example\XmlExample;
+use Findologic\MagentoExport\Export;
```
Comment (dblaichinger) [inline Findologic/Search/Controller/Export/ExportController.php:6]: Why is it called MagentoExport?

**Review by dblaichinger — CHANGES_REQUESTED** [review]

File: Findologic/Search/Controller/Export/ExportController.php (hunk 2 above)
Comment (TheKeymaster) [reply Findologic/Search/Controller/Export/ExportController.php:114]: Yes in Magento 2 it is per-vendor.

File: Findologic/Search/Controller/Export/ExportController.php (hunk 3 above)
Comment (TheKeymaster) [reply Findologic/Search/Controller/Export/ExportController.php:6]: To be clear it is for magento and it is the export.

Comment (dblaichinger) [reply Findologic/Search/Controller/Export/ExportController.php:6]: But where are here at `plugin-magento-2-di` and the plugin runs in Magento, so this should be clear.
Can we just go with `Findologic\Export'?

Comment (TheKeymaster) [reply Findologic/Search/Controller/Export/ExportController.php:6]: > Can we just go with Findologic\Export?

Well.. what if I told you that `libflexport` already uses that namespace:

image

Comment (dblaichinger) [reply Findologic/Search/Controller/Export/ExportController.php:6]: Ok leave it as it is :)

**Review by dblaichinger — APPROVED** [review]

**Pull request merged**